htdocs/compta/bank/releve.php

```diff
@@ -52,19 +52,18 @@
 if ($user->societe_id) $socid=$user->societe_id;
 $result=restrictedArea($user,'banque',$fieldid,'bank_account','','',$fieldname);
 
-if ($user->rights->banque->consolidate && $action == 'dvnext' && ! empty($dvid))
+if ($user->rights->banque->consolidate && ! empty($dvid))
```

Comment:
If we call the page with no action or any action, we will execute this code when we should not, wouldn't we ?
We should add test if $action start with dv for example.